src/par_iter/test.rs

```diff
@@ -352,6 +367,21 @@ pub fn check_partial_cmp_to_seq() {
 }
 
 #[test]
+pub fn check_partial_cmp_rng_to_seq() {
+    use rand::{Rng, SeedableRng, XorShiftRng};
+
+    let mut rng = XorShiftRng::from_seed([9346, 26355, 87943, 28346]);
+    let a: Vec<i32> = rng.gen_iter().take(1024).collect();
+    let b: Vec<i32> = rng.gen_iter().take(1024).collect();
+    for i in 0..a.len() {
+        let par_result = a[i..].par_iter().partial_cmp(b[i..].par_iter());
```

Comment:
Just FYI, the second call to `par_iter` (and `iter`) is not necessary: `a[i..].par_iter().partial_cmp(&b[i..])` is equivalent. But this is also fine, no need to change it -- more explicit this way.